internal/services/cockpit/types.go

```diff
@@ -18,6 +18,10 @@ var scopeMapping = map[string]cockpit.TokenScope{
 	"write_traces":        cockpit.TokenScopeWriteOnlyTraces,
 }
 
+func createGrafanaURL(projectID string) string {
+	return fmt.Sprintf("https://%s.dashboards.obs.fr-par.scw.cloud", projectID)
```

Comment:
Can the region be also passed? Otherwise I think we will have to change it down the road, it can be configured by default with `fr-par`

Reply:
Done